src/ray/raylet/node_manager.cc

```diff
@@ -542,6 +548,53 @@ ray::Status NodeManager::RegisterGcs() {
         "NodeManager.deadline_timer.print_event_loop_stats");
   }
 
+  if (RayConfig::instance().use_ray_syncer()) {
+    // Register resource manager and scheduler
+    ray_syncer_.Register(
+        /* component_id */ syncer::RayComponentId::RESOURCE_MANAGER,
+        /* reporter */ &cluster_resource_scheduler_->GetLocalResourceManager(),
+        /* receiver */ this,
+        /* upward_only */ false,
```

Comment:
why is this not true? raylet should only send update to the gcs?

Reply:
If true, this means, we'll not do broadcasting. Basically, it'll only be sent to GCS and GCS will not send it to raylet. Btw, I'm removing this.